src/Stratis.Bitcoin.Networks/StratisMain.cs

```diff
@@ -75,7 +76,7 @@ public StratisMain()
                 [BuriedDeployments.BIP66] = 0
             };
 
-            var bip9Deployments = new BIP9DeploymentsArray();
+            var bip9Deployments = new StratisBIP9Deployments();
```

Comment:
So did we ever have deployments for Stratis before?
How come we had `var bip9Deployments = new BIP9DeploymentsArray();` for stratis and something like
```
           var bip9Deployments = new BIP9DeploymentsArray
            {
                [BIP9Deployments.TestDummy] = new BIP9DeploymentsParameters(28, 1199145601, 1230767999),
                [BIP9Deployments.CSV] = new BIP9DeploymentsParameters(0, 1462060800, 1493596800),
                [BIP9Deployments.Segwit] = new BIP9DeploymentsParameters(1, 1479168000, 1510704000)
            };

```
for bitcoin?

Reply:
So we investigate and saw that even in StratisX, we don't have CSV activated.
So I guess it's all good. Or is it?

Reply:
Yes I was wondering the same. It seems we had no deployment parameters for Stratis before.